src/DynamoCore/Migration/Migration.cs

```diff
@@ -1360,6 +1354,7 @@ public class NodeMigrationAttribute : Attribute
         /// <summary>
         /// Version this migrates to.
         /// </summary>
+        [Obsolete("All migrations are now processed, beginning with the first migration whose version is >= the workspace version. The To property will be ignored.", false)]
```

Comment:
I do not believe that Zero Touch migrations are affected, because they don't use the `NodeMigrationAttribute`.